src/server/game/Entities/Creature/CreatureGroups.cpp

```diff
@@ -195,10 +195,13 @@ void CreatureGroup::FormationReset(bool dismiss)
     {
         if (itr->first != m_leader && itr->first->IsAlive())
         {
-            if (dismiss)
-                itr->first->GetMotionMaster()->Initialize();
-            else
-                itr->first->GetMotionMaster()->MoveIdle();
+            if (initiMotionMaster == true)
```

Comment:
don't need the `== true`

Reply:
im used to write like that. i dont recall the standards on the wiki prohibiting it. do i gotta remove it?

Reply:
is it against cpp standards? wiki doesnt say much about how to write this

Reply:
no, it's not really, but it is also unnecessary to specify as if it exists it will be true